README.md

```diff
@@ -2,6 +2,8 @@
 
 Apache Pulsar manager is a web-based GUI management tool for managing and monitoring Pulsar.
 
+[![FOSSA Status](https://app.fossa.io/api/projects/git%2Bgithub.com%2Fstreamnative%2Fpulsar-manager.svg?type=shield)](https://app.fossa.io/projects/git%2Bgithub.com%2Fstreamnative%2Fpulsar-manager?ref=badge_shield)
```

Comment:
Thank you very much for your contribution.
What do you think of putting the badge at the top of the content, similar to the following
![image](https://user-images.githubusercontent.com/1907867/64943823-4f19c780-d89f-11e9-8eae-dc17aab1081b.png)

Reply:
OK, I shall make another commit.
But should I put this badge below or beyond the chapter line of Apache Pulsar manager?
![image](https://user-images.githubusercontent.com/5338550/64947381-6f4d8480-d8a7-11e9-93f3-3c260472c9ba.png)

Reply:
I think this is better

![image](https://user-images.githubusercontent.com/1907867/64947613-fa2e7f00-d8a7-11e9-85ae-32cc93b987a6.png)

Reply:
Yep, agree with that.